### ISSUE TITLE
readthedocs docs are broken

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#83]
https://readthedocs.org/projects/eli5/builds/

**Comment by kmike** [comment]
It fails because of https://github.com/rtfd/readthedocs.org/issues/2504. 

I've also tried to use Python 2 virtualenv, but scipy installation fails in it because it seems pip/setuptools are old and don't support wheels.

Without virtualenv we need to mock every module in install_requires.

**Comment by lopuhin** [comment]
Is it possible to update pip and wheel first for the python 2 virtualenv?

**Comment by kmike** [comment]
A good call, but it turns out setuptools and pip are recent enough. I'm not sure why isn't it using a wheel for scipy: https://readthedocs.org/projects/eli5/builds/4667199/

**Comment by lopuhin** [comment]
I think putting scipy and other wheel packages in ``requirements.txt`` should help - I just checked locally, ``python setup.py install --force`` downloads a ``.zip`` for scipy, but pip installing it downloads a wheel. Installation from requirements.txt occurs in the previous step.

**Comment by kmike** [comment]
It helped, a great idea!
